### PR TITLE
Refactored move, userName()

### PR DESCRIPTION
**Pull request by rhfogh**
Replaced def move with def _set_value
Replaced move calls with set_value calls
Replaced userName() calls with use of username attribute
Replaced zoom_supported with read_only

A few mockups and complex motors still not complete.

### CODE REVIEW

File: HardwareObjects/EMBL/TINEMotor.py [hunk 1]
```diff
@@ -147,29 +147,16 @@ def abort(self):
...
-        :param target: float
-        :param wait: int
-        :param timeout: boolean
+        :param value: float
         :return:
         """
         if self.chan_state is not None:
             self.update_state(self.STATES.BUSY)
             self.chan_state.setOldValue("moving")
-        if target == float("nan"):
```
Comment (IvarsKarpics) [inline HardwareObjects/EMBL/TINEMotor.py]: could you please revert this if?

File: HardwareObjects/EMBL/EMBLPiezoMotor.py [hunk 2]
```diff
@@ -47,21 +47,12 @@ def init(self):
...
-        :type target: float
-        :param wait: wait finishing the move
-        :type wait: bool
-        :param timeout: timeout is seconds
-        :type timeout: float
+        :param value: target position
+        :type value: float
         :return: None
         """
-        if target == float("nan"):
```
Comment (IvarsKarpics) [inline HardwareObjects/EMBL/EMBLPiezoMotor.py]: Could you pleas revert the if

Comment (IvarsKarpics) [reply HardwareObjects/EMBL/EMBLPiezoMotor.py]: Ah ignore this... I see the check in the AbstractMotor

File: HardwareObjects/EMBL/TINEMotor.py (hunk 1 above)
Comment (IvarsKarpics) [reply HardwareObjects/EMBL/TINEMotor.py]: same here

File: HardwareObjects/EMBL/EMBLPiezoMotor.py (hunk 2 above)
Comment (rhfogh) [reply HardwareObjects/EMBL/EMBLPiezoMotor.py]: Just out of interest, I am not sure the original would have worked - I think that 'nan == nan' evaluates to False. But anyway, it is in AbstractMotor, as you say..

**Comment by marcus-oscarsson** [comment]
Looks good to me ! :+1:

**Comment by rhfogh** [comment]
With two approvals that ought to be mergeable. but

1) Can aonyone guess why the Codacy check is still hanging - the commit 
was three days ago?

2) Who should be doing the merging?

**Comment by marcus-oscarsson** [comment]
I think we should discuss the usage of Codacy and see if we cant replace it with something else. Maybe a black pre-commit hook and pylint would be more useful ?

I wanted to give the other sites an opportunity to have a look at the sites specific changes before I merged.

**Comment by rhfogh** [comment]
@marcus-oscarsson 
Makes sense. I just did not want it to fall between the cracks, but clearly you have it in hand.

**Pull request merged**